docs/02.deploying/01.production/04.internal/04.internal.md

```diff
@@ -244,6 +249,34 @@ kubectl rollout status ds neuvector-enforcer-pod
 ```
 
 + Make sure the console can be accessed and controllers are all online.
+</TabItem>
+<TabItem value="regenarate_cert_from_builtin" label="Regenarate certificate when built-in certificate is being used">
+If you didn't replace internal certificate before and want to migrate to a new set of certificate, follow the steps below:
+
++ Check if you already have internal certificate generated automatically.
+
+```bash
+kubectl get secret internal-cert -o yaml
+```
+
+If you see `tls.key`, `tls.crt` and `ca.crt` there, that means you've been using automatically generated certificate and you can skip this section.
+
+If you can see the secret, but cannot find these secrets, consider enabling `internal.autoRotateCert` in helm charts override. This option will generate and rotate your internal certificate automatically.
+
+If you don't use automatically generated internal certificate and can't do so, follow the steps below:
+
++ Follow the steps in `New certificate` tab to use kubernetes secret to manage internal certificate.  Instead of generating a new certificate, use the certificate, `old-ca.crt`, `old-tls.crt` and `old-tls.key`, retrieved below:
```

Comment:
```suggestion
+ Follow the steps in the `New certificate` tab to use a Kubernetes secret to manage the internal certificate. Instead of generating a new certificate, use the certificate, `old-ca.crt`, `old-tls.crt` and `old-tls.key`, retrieved below:
```